### DISCUSSION TITLE
Add backup volume to existing AIO instance ?

### DISCUSSION THREAD

**Discussion by T13nou** [nextcloud/all-in-one#3856] [Questions] [answered]
Hello !

I have a running instance of AIO since several months ago and as more people in my family are using it I'm considering adding the backup feature.

I have mounted a smb share in my host and I now need to link it to AIO.

Is there a non destructive way of doing that or do I need to recreate a new AIO instance with the updated volume parameters ?

Thanks,

T13nou

**Comment by szaimen** [comment, marked as answer]
Hi, can you post a screenshot of what you currently see in the AIO interface?
